src/sass/_common/common.scss

```diff
@@ -588,6 +588,10 @@ ul.bullet.gr-row > li {
     display: list-item;
 }
 
+.toggle-arrow#tax_information_note_toggle {
+    font-weight: bold;
+}
+
 .toggle-arrow {
```

Comment:
```suggestion
.toggle-arrow {
    &#tax_information_note_toggle {
        font-weight: bold;
    }
```

Reply:
do we need the toggle-arrow class mentioned at all? usually we prefer single identifiers, not grouping them together

Reply:
solved